src/ui/src/js/controllers/admin_garden.js

```diff
@@ -244,11 +244,17 @@ export default function adminGardenController(
 
     if (isReceiving){
       if (connection.config["subscribe_destination"] !== undefined){
-        url = url + " " + connection.config["subscribe_destination"];
+        if (connection.config["subscribe_destination"].charAt(0) != "/" && url.charAt(url.length - 1) != "/"){
```

Comment:
What about "/dest" and "url/"? This won't add a / to the end of url but will combine it into "url//dest". Is this possible?

Would it be better to strip any leading "/" from dest and any tailing "/" from url and manually add the "/" when doing the combine?

Reply:
dest = dest.replace(/^\/+/, "")
url = url.replace(/\/+$/, "")
url = url.concat("/", dest)

Reply:
I tried giving this a try and the regex wasn't supported in the way it was written. So I broke it down to a simple example for testing:
```javascript
const topic = "/topic/subtopic";
const regex = /^/+/;
console.log(topic.replace(regex, ''));
// Expected output: "topic/subtopic"
```

Reply:
Sorry, the escapes were lost because I didn't use the code feature. Fixed it.